runtime/dom-particle.js

```diff
@@ -97,6 +97,7 @@ class DomParticle extends XenStateMixin(Particle) {
   }
   _update(props, state) {
     this.config.slotNames.forEach(s => this.render(s, ["model"]));
+    this.relevance = this._shouldRender(this._props, this._state) ? 1 : -1;
```

Comment:
TODO here? Eventually this signal seems not strong enough (or too strong?), e.g. if we don't render the favorite food annotation for just one of many restaurants.

Reply:
done
i consider all the relevance related stuff under TODO, but one explicit extra todo certainly won't hurt :)